src/services/users/getUserById.js

```diff
@@ -0,0 +1,9 @@
+const getUser = async id => {
+  const response = await fetch(`${process.env.REACT_APP_API_URL}/users/${id}`);
+  return response.json();
+};
+
+export const getUserById = async id => {
+  const data = await getUser(id);
+  return data;
```

Comment:
Fun fact, if you define a method as `async`, it will never return a promise. So you can instead do: 
```suggestion
  return getUser(id);
```

This will then await `getUser` and return the result

Reply:
I've made the updates as suggested in commit [d06e633](https://github.com/SheCodesAus/she-codes-group-project-frontend-fullstack-pancake/commit/d06e633b3bf15f3fd0230c908e6bcedeb7812821), but I still don't understand this one, what's the difference..